packages/projects/src/views/Browse.jsx

```diff
@@ -45,6 +45,32 @@ const humanize = (ms, dp = 0) => {
   return `${scaledTime.toFixed(dp)} ${timeUnits[timeScalarIndex]}`
 }
 
+const sortByUpdated = items => Object.values(items).sort(compareByUpdated)
```

Comment:
fyi: for numeric comparison, you can simplify

```
Object.values(items).sort((a, b) => b - a)
```

I can't add a comment directly, but while you're at it, maybe we can change `humanize` to not show the (s) but be smarter. I changed it in the explorer here:

https://github.com/kernel-community/services/blob/main/packages/explorer/src/views/Browse.js#L38

Probably a good idea to think about some common utils namespace to share these across.